README.md

```diff
@@ -0,0 +1,32 @@
+# DevHelp
+
+DevHelp doesn't come with documentation.
+
+You can install documentation from your package manager or from Flatpak.
+
+## Package managers
+
+DevHelp will automatically find and watch for documentation installed on the host.
+
+Fedora example
+
+```sh
+sudo dnf install gtk4-devel-docs glib2-doc libadwaita-doc libportal-devel-doc libsoup3-doc  webkitgtk6.0-doc
+```
+
+Ubuntu example
+
+```sh
+sudo apt install libgtk-4-doc libglib2.0-doc libadwaita-1-doc libportal-doc
+```
+
+## Flatpak
+
+When consulting documentation installed from Flatpak, you will need to start DevHelp with a special command.
+
+```sh
+# Install GNOME 44 docs
+flatpak install org.gnome.Sdk.Docs//44
+# Start DevHelp for GNOME 44 docs
+flatpak run --nofilesystem=host --runtime-version=44 org.gnome.Devhelp
```

Comment:
This sis wrong, overriding the runtime version won't work the moment there's an ABI break. We don't need to hardcode the version number either, Devhelp will pick up the .Docs extension corresponding to the version it was built against.